packages/debug/src/browser/debug-prefix-configuration.ts

```diff
@@ -49,15 +50,30 @@ export class DebugPrefixConfiguration implements CommandContribution, CommandHan
     @inject(LabelProvider)
     protected readonly labelProvider: LabelProvider;
 
+    @inject(StatusBar)
+    protected readonly statusBar: StatusBar;
+
     readonly prefix = 'debug ';
     readonly description = 'Debug Configuration';
+    readonly statusBarId = 'select-run-debug-statusbar-item';
 
     private readonly command: Command = {
         id: 'select.debug.configuration',
         category: 'Debug',
         label: 'Select and Start Debugging'
     };
 
+    @postConstruct()
+    init(): void {
```

Comment:
protected

Reply:
Updating the method to protected causes issues with the compatibility of `QuickOpenHandler` which is unprotected.